### PR TITLE
Remove xx_sys to xx_ffi renaming

### PR DESCRIPTION
**Pull request by EPashkin**
Part of https://github.com/gtk-rs/gir/issues/685

cc @GuillaumeGomez, @sdroege

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
👍

**Comment by EPashkin** [comment]
Updated too

**Comment by sdroege** [comment]
Why is this WIP? Good to go in any case with the latest version of the gir PR.

**Comment by sdroege** [comment]
gir PR is merged now, so let's get this updated and merged? :)

**Comment by EPashkin** [comment]
Updated

File: src/application.rs [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-use ffi;
-use glib::GString;
+use gio_sys;
 use glib::object::Cast;
 use glib::object::IsA;
-use glib::signal::{SignalHandlerId, connect_raw};
+use glib::signal::{connect_raw, SignalHandlerId};
 use glib::translate::*;
-use glib_ffi;
+use glib::GString;
```
Comment (sdroege) [inline src/application.rs]: How did you get this all re-ordered, if not by rustfmt... but if rustfmt, why isn't everything else nicely formatted too then? :)

**Comment by sdroege** [comment]
:+1:

File: src/application.rs (hunk 1 above)
Comment (EPashkin) [reply src/application.rs]: I use rustfmt but stashed only this part 😉

Comment (sdroege) [reply src/application.rs]: Maybe it's soon time to switch to it completely then ;) Anyway, fine with me to only take this change for now :)

**Comment by EPashkin** [comment]
CI "passed"

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**